File: app/src/main/java/com/eason/MainActivity.java
package com.eason;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import com.eason.animation.ETranslateActivity;
import com.eason.touchevent.ETouchEventActivity;

public class MainActivity extends BaseActivity {

    @Override
    protected void init() {

    }

    @Override
    public int getLayoutId() {
        return R.layout.activity_main;
    }

    /**
     * 动画
     *
     * @param view
     */
    public void animation(View view) {
        startActivity(new Intent(this, ETranslateActivity.class));
    }

    public void touchEvent(View view) {
        startActivity(new Intent(this, ETouchEventActivity.class));
    }
}

File: app/src/main/java/com/eason/MainActivity.kt
package com.eason

import android.content.Intent
import android.view.View
import com.eason.animation.ETranslateActivity
import com.eason.touchevent.ETouchEventActivity

class MainActivity : BaseActivity() {

    override val layoutId: Int
        get() = R.layout.activity_main

    override fun init() {

    }

    /**
     * 动画
     *
     * @param view
     */
    fun animation(view: View) {
        startActivity(Intent(this, ETranslateActivity::class.java))
    }

    fun touchEvent(view: View) {
        startActivity(Intent(this, ETouchEventActivity::class.java))
    }
}
